helm-charts/seldon-core-operator/README.md

```diff
@@ -13,7 +13,7 @@ helm repo add seldonio https://storage.googleapis.com/seldon-charts
 helm repo update
 ```
 
-Once that's done, you should then be able to deploy the chart as:
+Onca that's done, you should then be able to deploy the chart as:
```

Comment:
:upside_down_face: I guess we have a typo in the base chart

Reply:
We can raise an issue for the next release :smiling_face_with_tear: